images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -115,6 +115,10 @@ function Get-YarnVersion {
     return "Yarn $yarnVersion"
 }
 
+function Get-PowershellCoreVersion {
```

Comment:
Please pay attention, you duplicate the function at the 64 line :
```
function Get-PowershellVersion {
    return $(pwsh --version)
}
```

Reply:
thank you, I've fixed this.